tests/conftest.py

```diff
@@ -28,15 +29,21 @@ def __init__(self, pixel_array, slice_position, row_cosine=None, column_cosine=N
         if column_cosine is None:
             column_cosine = y_cos
 
-        na, nb = pixel_array.shape
+        shape = pixel_array.shape
+        if len(shape) == 2:
+            na, nb = shape
+            SamplesPerPixel = 1
+        else:
+            na, nb, SamplesPerPixel = shape
 
         self.pixel_array = pixel_array
 
         self.SeriesInstanceUID = 'arbitrary uid'
         self.SOPClassUID = 'arbitrary sopclass uid'
         self.PixelSpacing = [1.0, 1.0]
-        self.Rows = na
-        self.Columns = nb
+        self.Columns = na
```

Comment:
Note that your code appears to swap `na` and `nb`. My suggestions here swap them back---I am assuming your swap was unintentional.

```suggestion
        self.Columns = num_columns
```

Reply:
👍